src/scss/components/checkbox/_config.scss

```diff
@@ -0,0 +1,6 @@
+/// ========== VARIABLES ========== ///
+@import "vars";
+
+/// ========== MIXINS ========== ///
+@import "mixins/hxCheckboxControl";
+@import "mixins/hxCheckboxFacade";
```

Comment:
Please prefix paths with `./` to be explicit that these are local mixins, not global.

Reply:
Paths prefixed paths with `./`